praetorian_cli/sdk/entities/account_discovery.py

```
@@ -0,0 +1,397 @@
+                return None
+
+            agents_data = resp.json()
+            if not agents_data:
+                return None
+
+            return _build_account_info(
+                account_email,
+                len(agents_data),
+                metadata,
+            )
+        except Exception as e:
```

Ruff (0.15.5)

[warning] Do not catch blind exception: `Exception`

(BLE001)